Dockerfile.cuda

```diff
@@ -4,15 +4,18 @@ ARG TARGETARCH
 ARG VERSION=0.0.0
 
 WORKDIR /go/src/github.com/jmorganca/ollama
-RUN apt-get update && apt-get install -y git build-essential cmake
+RUN apt-get update && apt-get install -y git build-essential cmake nvidia-cuda-toolkit
```

Comment:
This shouldn't be necessary since this is using the nvidia/cuda devel image as the base image